assignments/this.js

```diff
@@ -1,26 +1,58 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Global Object Binding - when this isn't specified as to what object it is attached to so it defaults to the console object which is all of JavaScript.
+* 2. Implicit Binding - This is when you call a function using a specified object in the form object.function(). This in this case refers to the object specified.
+* 3. New Binding - This case is used when using a constructor to make a new object and 'this' will refer to that specific instance of that object being created.
+* 4. Explicit Binding - This usage appears when using .apply/.call to override what the constructor made this refer to and instead to look at a specified instance.
 *
 * write out a code example of each explanation above
 */
 
 // Principle 1
-
 // code example for Window Binding
+console.log("Window Binding");
+function addTwo(a,b){
```

Comment:
So long as you understand that nothing is being done with the arguments being passed in, this is totally fine.